pom.xml

```diff
@@ -33,4 +33,28 @@
 			<version>1.4.3</version>
 		</dependency>
 	</dependencies>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<version>3.3.0</version>
```

Comment:
There is much newer version of shade-plugin, 3.6.0. Maybe we can use the newest version?
![image](https://github.com/user-attachments/assets/8c838cf1-e163-4165-ad45-a1a81fbd328f)

Reply:
Fixed